pkg/kv/kvserver/rangefeed/budget.go

```diff
@@ -16,11 +16,21 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/rpc"
+	"github.com/cockroachdb/cockroach/pkg/util/envutil"
 	"github.com/cockroachdb/cockroach/pkg/util/mon"
 	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/errors"
 )
 
+// Environment variable controlling rangefeed budgets. If false budgets are
+// disabled otherwise enabled.
+const rangeFeedBudgetEnvVarName = "COCKROACH_USE_RANGEFEED_MEM_BUDGETS"
```

Comment:
nit: you can just move the variable itself up here, since envvars don't change:

```go
var rangeFeedBudgetEnabled = envutil.EnvOrDefaultBool()
```